client/extensions/woocommerce/app/dashboard/store-move-notice-view.js

```diff
@@ -12,7 +12,7 @@ import { localize, translate } from 'i18n-calypso';
  */
 
 import { Card, Button } from '@automattic/components';
-import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import { getSelectedSiteWithFallback } from 'calypso/state/sites/selectors';
```

Comment:
This selector is essentially a copy of the one in here, so we can use this one instead.

Reply:
We could get rid of `getSelectedSiteWithFallback` completely and follow the Calypso convention. Which is to always have a site in the URL if one is selected. And for users with a single site, a path like `/home` is auto-redirected to `/home/primary.blog` by the [`siteSelection` handler](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/controller.js#L413-L434).

Reply:
That would sound like work for another PR I guess. Also, this selector already exists and is being used, so definitely don't see a reason to refactor away from it here.

Reply:
Sounds good, am already working on it 😄

Reply:
Thanks a bunch 🙌